### PR TITLE
🤖: Update build recipes for SP3

### PR DESCRIPTION
**Pull request by dcermak**
This is an automated pull request to update from branch for-deploy-3

### CODE REVIEW

File: .obs/workflows.yml [hunk 1]
```diff
@@ -65,3 +65,60 @@ workflow:
...
+        package: pcp-image
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: rmt-server
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: golang-1.18
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: ruby-2.5-image
```
Comment (dirkmueller) [inline .obs/workflows.yml:93]: that should be triggered against SP4, no?

File: .obs/workflows.yml [hunk 2]
```diff
@@ -65,3 +65,60 @@ workflow:
...
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: nodejs-14
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: nodejs-16
```
Comment (dirkmueller) [inline .obs/workflows.yml:99]: same

File: .obs/workflows.yml [hunk 3]
```diff
@@ -65,3 +65,60 @@ workflow:
...
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: openjdk-11-devel
```
Comment (dirkmueller) [inline .obs/workflows.yml:102]: same

File: .obs/workflows.yml [hunk 4]
```diff
@@ -65,3 +65,60 @@ workflow:
...
         source_package: micro
         target_project: home:defolos:BCI:CR:SLE-15-SP3:Staging
+  filters:
+    event: pull_request
+
+refresh_devel_BCI:
+  steps:
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: python-3.6
```
Comment (dirkmueller) [inline .obs/workflows.yml:75]: that should only be tested on SP4?

File: .obs/workflows.yml (hunk 1 above)
Comment (dcermak) [reply .obs/workflows.yml:93]: No, this is just telling obs to pull from git. What we do with the package result is not related to this.

**Review by fcrozat — APPROVED** [review]

File: .obs/workflows.yml (hunk 1 above)
Comment (dirkmueller) [reply .obs/workflows.yml:93]: where can we remove those containers from the  sle15-sp3 branch?

Comment (dcermak) [reply .obs/workflows.yml:93]: Don't create any `BaseContainerImage` instances for 15 SP3 anymore and `git rm` everything that you don't like from the `sle15-sp3` branch.

**Pull request merged**